starter-code/views/search-form.ejs

```diff
@@ -0,0 +1,8 @@
+<body>
+  <form action="search" method="POST">
+    <label for="joke">
+      <input id="joke" type="text" name="jokeKeyword">Search a joke</input>
```

Comment:
We usually don't put things inside of an `input` tag. The proper what do to it would look like this:

```
<label>
      Search a joke
      <input type="text" name="jokeKeyword">
 </label>
```

Or:

```
<label for="joke>
      Search a joke
</label>
<input if="joke" type="text" name="jokeKeyword">
```